### ISSUE TITLE
GestureDetector blocking ScrollView in Mobile Web (even if explicitly disabled)

### ISSUE THREAD

**Issue by mikehobi** [software-mansion/react-native-gesture-handler#3010]
### Description

I'm trying to create a scrolling feed with a draggable/swipable elements.

When I include `GestureDetector` inside a `ScrollView`, scrolling is disabled altogether. I'm trying to implement a solution that allows vertical scrolling and horizontal swiping (on the draggable elements)

The solutions appear to work for iOS/Android, but not for mobile web.

I've provided a simple repro'd snack demo.

### Steps to reproduce

1. Implement GestureRoot, ScrollView, and Gesture components
2. Add `Gesture.Pan`, set to `enabled(false)` or `manualActivation`
3. See that scrolling is blocked on Safari (or when emulating mobile device in Chrome Devtools)

### Snack or a link to a repository

https://snack.expo.dev/@mikehobi/awkward-red-cereal?platform=web

### Gesture Handler version

2.16.1

### React Native version

0.74.3

### Platforms

Web

### JavaScript runtime

None

### Workflow

Expo bare workflow

### Architecture

None

### Build type

None

### Device

Real device

### Device model

iPhone Latest

### Acknowledgements

Yes

**Comment by m-bert** [comment]
Hi @mikehobi! This issue seems closely related to #2617. You can try to add [touchAction](https://docs.swmansion.com/react-native-gesture-handler/docs/gestures/gesture-detector/#touchaction-web-only) property to `GestureDetector` and set it to `pan-y`. This is already done internally in our [Swipeable](https://github.com/software-mansion/react-native-gesture-handler/blob/7050c1b2571a9f6a8213eba0c909b0bcee216a75/src/components/Swipeable.tsx#L544) implementation. Let me know if it helps!

**Comment by mikehobi** [comment]
Thanks, touch-action does indeed solve my issue.

I guess I am still curious why `.enabled(false)` still captures events though, seems as though if it's disabled completely, it should not block other events.

Closing!

**Comment by m-bert** [comment]
> I guess I am still curious why .enabled(false) still captures events though, seems as though if it's disabled completely, it should not block other events.

When handler is created, it sets `touch-action: none;` on underlying view by default. Currently there's no logic that checks if handler is enabled or not, so changing this property won't change `touch-action`.

Do you think it should be added?
